core/arch/arm/kernel/generic_boot.c

```diff
@@ -534,6 +534,19 @@ void *get_external_dt(void)
 
 static void release_external_dt(void)
 {
+	int ret;
```

Comment:
could you update this to init the local variable? (https://optee.readthedocs.io/general/coding_standards.html?highlight=All%20variables%20shall%20be%20initialized)